dev/devicelab/bin/tasks/post_backdrop_filter_perf__timeline_summary.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter_devicelab/tasks/perf_tests.dart';
+import 'package:flutter_devicelab/framework/adb.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+
+Future<void> main() async {
```

Comment:
Considering that our devicelab's resources are currently very limited, maybe we don't have to add the Android test until we find good use of it. I think with or without your optimization, the performance on Android should be similar?

Reply:
That makes sense, I'll also remove the entry from the manifest.